Makefile

```diff
@@ -8,7 +8,7 @@ all: test
 
 .PHONY: black_check
 black_check:
-	if python -c 'import sys ; sys.exit(1 if sys.version.startswith("2.") else 0)' ; then black --check testslide/ tests/ ; fi
+	if command -v black; then black --check testslide/ tests/ ; fi
```

Comment:
There was already a conditional to require black for supported python versions at setup.py, plus this conditional here, any reason we can't keep that?

Reply:
Writing the conditional to support black requires checking the python version is at least 3.6. It got to be a lot longer to fully write out that check, so I just decided to check and see if black was installed. That seems like a more reasonable check, plus we ensure on CI that it's installed. I would argue if the user has black provided via some other means, you probably want this target to succeed.